### ISSUE TITLE
'numpy.ndarray' object has no attribute 'normalizing_scale'

### ISSUE THREAD

**Issue by happyle1004** [JakeColtman/bartpy#51]
If I use:
model = SklearnModel()
model.fit(X_train, y_train.to_numpy())
I got 'numpy.ndarray' object has no attribute 'normalizing_scale' this error message. 

If I use:
model = SklearnModel
model.fit(X_train, y_train.to_numpy())
I got 'fit() missing 1 required positional argument: 'y'' this error message. 

X_train is a pandas dataframe.

**Comment by shreyaskar123** [comment]
Has anyone resolved this yet? I am facing the same issue.

**Comment by ahmedlebo** [comment]
In the data.py file i changed line 217 
from:
self.original_y_min, self.original_y_max = y.min(), y.max()

to:
self.original_y_min, self.original_y_max = np.min(y),np.max(y)

and it works for me
![Screen Shot 2021-10-30 at 11 22 27 PM](https://user-images.githubusercontent.com/44555496/139558854-9c9d8176-6673-4fb7-8627-0629d3d158b2.png)

**Comment by emc5ud** [comment]
If anyone still needs help with this, I installed this branch: 

```
pip install git+https://github.com/JakeColtman/bartpy.git@pytorch --upgrade
```

And the error no longer appeared.

**Comment by watson-sam** [comment]
Any chance of getting this fix @emc5ud into the master branch / release code? Cheers
